### PR TITLE
test: AddGenerated Type Test in PrettierIgnore

### PR DESCRIPTION
**Pull request by jikim-msft**
#### Description

This PR makes prettier to ignore enforcing its rule on `generated.ts`

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -16,3 +16,6 @@ common/lib/**/lib
 **/_api-extractor-temp/*
 **/api-report/*
 **/*.api.md
+
+# End-to-End Test
```
Comment (Josmithr) [inline .prettierignore:20]: I would update these comments to say something like 
```suggestion
# Generated type-tests
```

Comment (Josmithr) [reply .prettierignore:20]: And I see a couple of entries where the comment is missing - I would advocate for adding it (even if there aren't other comments in those files)

Comment (Josmithr) [reply .prettierignore:20]: Looks like you missed this one. But the others look good :)

Comment (jikim-msft) [reply .prettierignore]: Thanks for the catch! 😄

**Review by tylerbutler — APPROVED** [review]

**Pull request merged**